src/client/refresh/soft/sw_polyset.c

```diff
@@ -299,6 +291,41 @@ quotient must fit in 32 bits.
 FIXME: GET RID OF THIS! (FloorDivMod)
 ====================
 */
+#if defined(__i386__) || defined(__amd64__)
+// just exclude ARM32 with FPU, e.g. Pi 1
+static void
+FloorDivMod(int numer, int denom, int *quo, int *rem)
```

Comment:
Could you please move `#ifdef` inside of function declaration? It should keep function definition in case if code will change in future and definition have changed only in one place by mistake.